### PR TITLE
Fixed bug for 430/431 Case Sensivity & Email

### PR DESCRIPTION
**Pull request by jamesohmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-430
https://tools.hmcts.net/jira/browse/RDCC-431

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/userprofileapi/client/UpdateUserProfileData.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
     private String idamStatus;
 
     @JsonCreator
-    public UpdateUserProfileData(@JsonProperty(value = "emailAddress") String email,
+    public UpdateUserProfileData(@JsonProperty(value = "email") String email,
```
Comment (AdilOoze) [inline src/main/java/uk/gov/hmcts/reform/userprofileapi/client/UpdateUserProfileData.java]: Any reason why this is being changed?

Comment (shreedhar-hmcts) [reply src/main/java/uk/gov/hmcts/reform/userprofileapi/client/UpdateUserProfileData.java]: This is just to be in sync with Quartz and there was also duplicate field emailAddress was appearing on Swagger

Comment (AdilOoze) [reply src/main/java/uk/gov/hmcts/reform/userprofileapi/client/UpdateUserProfileData.java]: Okay great!

**Comment by AdilOoze** [comment]
Is this meant to be raised into Master?

**Pull request merged**